### ISSUE TITLE
Do not close the project when exporting

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#1628]
It is annoying to have to reload the project after exporting it.

**Comment by cdwhitfield73** [comment]
There were several issues with the exporting process that I fixed along with this.  The reason it closed the project was because it had to open the project as well (if you export a project that isn't open) just to get the manifest data.  I removed the need to open the project at all by reading the manifest directly.  Now, exporting does not affect any project that is open at all.  I also added the "in progress" alert, but only after the user chooses the save location, which meant reorganizing the way the function worked.  I created a global way to call the "Save as" dialog in the main.js.  Also, this function was mixing Promises with asynchronous callbacks in a way that could create problems.  Rebuilt the entire function into one clean Promise chain.  Also added more specific error handling messages.  End result, the exporting function still works well and the project is neither opened or closed in the process.  An in progress indicator comes up (although very briefly on a fast computer) and success or fail messages are given to the user.
